Java/src/main/java/Planes/Plane.java

```diff
@@ -3,58 +3,58 @@
 import java.util.Objects;
 
 abstract public class Plane {
```

Comment:
Although it is allowed to have access and non-access modifiers in different order, in real world it is a bad style. The convention is to declare them as follows
`public protected private abstract default static final transient volatile synchronized native strictfp`

References
https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers
https://docs.oracle.com/javase/specs/jls/se11/html/jls-8.html#jls-8.1.1